syntax = "proto3";

message AutoSearchFromDtu {
  string dtu_sn = 1;
  int32 time = 2;
  int32 package_count = 3;
  int32 package_idx = 4;
  repeated int64 mi_sns = 5;
}

message AutoSearchToDtu {
  string ymd_hms = 1;
  int32 offset = 2;
  int32 package_idx = 3;
  int32 err_code = 4;
  int32 time = 5;
}
